src/plugins/telemetryTable/TelemetryTableViewProvider.js

```diff
@@ -51,6 +51,14 @@ define([
             view(domainObject, objectPath) {
                 let table = new TelemetryTable(domainObject, openmct);
                 let component;
+
```

Comment:
This should likely be part of a different branch and PR? I don't think it's specific to Conditionals so if it's required to go out sooner, it could.